ocfweb/login/templates/login/ocf/login.html

```diff
@@ -19,9 +19,12 @@
                     {% csrf_token %}
                     <div style="max-width: 300px;">
                         {{ form|bootstrap }}
+            <small class="reset-link">
+                <a href="{% url 'change_password' %}">Forgot your password?</a>
+            </small>
```

Comment:
indented weird (plus, hard tabs?)

Reply:
Hm, that is strange I'll fix that. When you say hard tabs, are you suggesting I use them in future or something else?

Reply:
yeah, we generally use 4 space indents at the ocf (unless the language has a well-known convention, e.g. overwhelming puppet convention is 2 space indents, so we use that)